muikku-persistence/src/main/java/fi/otavanopisto/muikku/model/users/UserSchoolDataIdentifier.java

```diff
@@ -15,65 +15,83 @@
 import fi.otavanopisto.muikku.model.base.SchoolDataSource;
 
 @Entity
-@Table (
-  uniqueConstraints = {
-    @UniqueConstraint (columnNames = { "dataSource_id", "identifier" })
-  }    
-)
+@Table(uniqueConstraints = { @UniqueConstraint(columnNames = { "dataSource_id", "identifier" }) })
```

Comment:
Any chance of archived/unarchived causing unexpected constraint violations?

Reply:
Hmm it probably could but I would almost consider them welcome as we probably don't want users with the same identifier in the database. Not sure tbh.